net/apfree-wifidog/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2015,2018 KunTeng.org.cn

```

Comment:
in fact apfree-wifidog was used since 2014 in apfree ROM, it was open source in 2015 and used by kunteng cooperation, here is apfree-wifidog open source url:
 https://github.com/liudf0716/apfree_wifidog

Reply:
The question isn't about the target application, it's about this makefile that you are claiming copyright on.

Reply:
ok，i have changed it. thx!